packages/editor/src/components/page-attributes/parent.js

```diff
@@ -249,12 +255,15 @@ export function ParentRow() {
 							onClose={ onClose }
 						/>
 						<div>
-							{
-								/* translators: The domain name should be a reserved domain name to prevent linking to third party sites outside the WordPress project's control. You may also wish to use wordpress.org or a wordpress.org sub-domain. */
+							{ sprintf(
+								/* translators: %1$s The home URL of the WordPress installation without the scheme. */
 								__(
-									"Child pages inherit characteristics from their parent, such as URL structure. For instance, if 'Web Design' is a child of 'Services', its URL would be example.org/services/web-design."
-								)
-							}
+									'Child pages inherit characteristics from their parent, such as URL structure. For instance, if "Pricing" is a child of "Services", its URL would be %1$s/services/pricing.'
+								),
+								homeUrl
```

Comment:
We just want the host, right?

The `@wordpress/url` methods `filterURLForDisplay( homeUrl )` or `getAuthority( homeUrl )` should do the same thing I think. Probably the latter.

Reply:
Added `filterURLForDisplay()` in bacc715e625477e3055ba136a20a3beca90df9b1. I used that as the latter doesn't include sub-directories: ie, make.wordpress.org/core/ would display as make.wordpress.org.